android/TodokTodok/app/src/main/java/com/team/todoktodok/data/datasource/discussion/DefaultDiscussionRemoteDataSource.kt

```
@@ -45,6 +46,21 @@ class DefaultDiscussionRemoteDataSource(
                 ),
         )
 
+    override suspend fun deleteDiscussion(discussionId: Long) {
+        discussionService.deleteDiscussion(discussionId)
+    }
+
+    override suspend fun toggleLike(discussionId: Long): LikeAction =
+        when (discussionService.toggleLike(discussionId).code()) {
+            201 -> LikeAction.LIKE
+            204 -> LikeAction.UNLIKE
+            else -> throw IllegalStateException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)